### ISSUE TITLE
Cannot read property 'getEditor' of null when component is unmounted

### ISSUE THREAD

**Issue by benjy** [ckeditor/ckeditor4-react#49]
## Are you reporting a feature request or a bug?

Bug

Could be related to #35 or #44 although they list different errors.

## Provide detailed reproduction steps (if any)

1. Navigating away with react-router
2. Causes unmount
3. Triggers an error in CKEditor

### Expected result

No error.

### Actual result

Error:
```
index.js:1437 TypeError: Cannot read property 'getEditor' of null
    at a (ckeditor.js:359)
    at Object.CKEDITOR.replace (ckeditor.js:363)
    at ckeditor.js:1692
```

## Other details

* Browser: Chrome
* OS: OSX
* Integration version: 1.0.0-beta
* CKEditor version: 4.14
* Installed CKEditor plugins: Not sure how to list all these, mainly everything from basic + image2 and auto grow

**Comment by Gajesh-LH** [comment]
+1 

I am facing the same issue any workaround on this.

![Screen Shot 2019-06-25 at 3 05 02 PM](https://user-images.githubusercontent.com/38095258/60087445-a9046880-975a-11e9-8894-17675876b9e2.png)

**Comment by f1ames** [comment]
I think this may get fixed after solving #44 (and merging #16).

**Comment by f1ames** [comment]
Fixed in https://github.com/ckeditor/ckeditor4-react/commit/7b34d2c4f896ced08e66359faca28194ed7e8ef4 via https://github.com/ckeditor/ckeditor4/pull/3128.
